index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>BagIt Profiles Specification 1.3.0</title>
+    <title>BagIt Profiles Specification 1.4.0</title>
```

Comment:
Would you mind adding my orcid and current institutional affiliation, while we're here?
- orcid: 0000-0002-9997-3464
- company: Lyrasis

Reply:
🙏🏽